presto-spi/src/main/java/com/facebook/presto/spi/type/VarcharType.java

```diff
@@ -39,6 +40,11 @@ public static VarcharType createVarcharType(int length)
         return new VarcharType(length);
     }
 
+    public static TypeSignature getParametrizedVarcharSignature(String param)
```

Comment:
typo, should be `getParameterizedVarcharSignature`. Also, I think the argument should be an `int` not a `String` to make the API safer